### ISSUE TITLE
Flat out doesn't work

### ISSUE THREAD

**Issue by darthpale** [canton7/SyncTrayzor#738]
First it couldn't find the instance of syncthing, then I found a setting for the syncthing directory set that and now synctrayzor doesns't start.  only reporting this issue because that's what the dialog box said to do.

Sorry if this is not in accord with  your guidelines, if they contradict your own in app directions.

![image](https://user-images.githubusercontent.com/48268993/202059471-3204d0cc-3829-4597-9b84-3a096e821384.png)

Version: 1.1.29.0; Variant: Installed; Arch: Amd64
Path: C:\Program Files\SyncTrayzor\SyncTrayzor.exe
System.UnauthorizedAccessException: Access is denied.

   at Pri.LongPath.File.Copy(String sourcePath, String destinationPath, Boolean overwrite)
   at SyncTrayzor.Services.FilesystemProvider.Copy(String from, String to)
   at SyncTrayzor.Services.Config.ConfigurationProvider.Initialize(Configuration defaultConfiguration)
   at SyncTrayzor.Bootstrapper.Configure()
   at Stylet.BootstrapperBase.Start(String[] args)
   at System.Windows.Application.OnStartup(StartupEventArgs e)
   at System.Windows.Application.<.ctor>b__1_0(Object unused)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

**Comment by canton7** [comment]
I'm not sure what you are trying to do, but you've presumably set Syncthing Custom Path to a non-writable directory?

SyncTrayzor copies syncthing.exe into AppData on first start. This directory needs to be writable, so that Syncthing can upgrade itself. If you've set Syncthing Custom Path to a non-writable directory, SyncTrayzor won't be able to copy syncthing into it, which is probably what you're seeing.

**Comment by darthpale** [comment]
appdata is writable.
There is a syncthing.exe in there (which synctrayzor must have put in there)
running that produces errors about the database being the wrong version.   It was not clear that your product included an out-of-date syncthing itself, and so when it tried to start it it couldn't.  

I manually copied the syncthing I had downloaded to your folder and reinstalled your product it seems to work OK now.  
Again I only opened the issue because that's what the dialog box your product produced said to do.  (You might want to re-think that advice in the dialog box)

Your advice from migrating from "raw" syncthing to synctrazor installed of doing nothing did not work for me. I had to manually copy syncthing.exe from my "raw" install location to yours in %APPDATA%.  (This copy worked without privilege escalation) My user is the owner of all the folders in question and has full-control permissions.)

This may only be the case if the "raw" syncthing  newer than what you include in the installer.   Your included syncthing is v1.18.1 current syncthing version from them is 1.22.1 

Does that auto-update of syncthing happen on first run?  Didn't seem to

Before copying my "raw" 1.22.1 syncthing to your %appdata% folder, I ran it from the terminal, and got the following error:
[start] 15:36:01 INFO: syncthing v1.18.1 "Fermium Flea" (go1.16.6 windows-amd64) teamcity@build.syncthing.net 2021-07-30 12:41:57 UTC
[start] 15:36:01 WARNING: Failed to initialize config: config file version (37) is newer than supported version (35). If this is expected, use -allow-newer-config to override.
[monitor] 15:36:01 INFO: Syncthing exited: exit status 1

Notice that your old included version 1.18.1 could not open the v37 config file from 1.22.1
